clients/src/main/java/org/apache/kafka/common/protocol/types/Struct.java

```diff
@@ -292,13 +292,15 @@ public String toString() {
             b.append('=');
             if (f.type() instanceof ArrayOf) {
```

Comment:
I guess that would drop into the else statement, which works but I am not sure is "better" of different enough to matter. 

Why would you prefer it in the first if check?